config/locales/zh-CN.yml

```diff
@@ -92,10 +92,12 @@ zh-CN:
         expires_at: "过期日期"
         name: "名称"
         path: "路径"
+        promotion_category: "促销分类"
```

Comment:
This key do not exists in Spree core `master` locale as a sub key (it do exist as a root key and as an ActiveRecord model). See: https://github.com/spree/spree/blob/master/core/config/locales/en.yml